Dockerfile

```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:1
 
-ARG GO_VERSION="1.20"
+ARG GO_VERSION="1.19"
```

Comment:
v6.4.x we user go version 1.20

Reply:
thank you. was gettin open64 error, using alpine3.18 fixes the error